app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Dict, Tuple
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    func_cache: Dict[Tuple, any] = {}
+
+    def wrapper(*args) -> Callable:
+        if args in func_cache:
+            print("Getting from cache")
```

Comment:
what if two different functions have the same `args` but different `kwargs`?
you must use `args` + `kwargs` as the keys of `func_cache`